examples/achromatic_spectrometer/input_spectrometer.in

```diff
@@ -8,10 +8,10 @@ beam.charge = 1.0e-9
 beam.particle = electron
 beam.distribution = waterbag
 beam.lambdaX = 3.162277660e-6
-beam.lambdaY = 3.162277660e-6
+beam.lambdaY = beam.lambdaX
```

Comment:
Do we really want to change all of the existing examples?  Couldn't this cause some confusion to users?  It might be better to have just one example to illustrate this feature.

Reply:
I added it whenever it made sense so people come definitely across it. I hope I did not add it in confusing places, can revert a few where it is complicating the logic ofc.